.github/ops-bot.yaml

```diff
@@ -17,7 +17,7 @@
 # - https://github.com/rapidsai/ops-bot
 
 auto_merger: true
-branch_checker: true
+branch_checker: false
```

Comment:
Why is this change being made?
I realize that for `release/1.2` it is currently failing, but:
1) I have a PR to fix this
2) We don't want this accidentally being forward-merged.